book/introduction.md

```diff
@@ -0,0 +1,66 @@
+# Introduction
+
+<img src="./innersource-patterns-book-cover.jpg" title="InnerSource Patterns">
+
+{% hint style="info" %}
+This is an early release of the InnerSource Patterns book.
+You may still find broken link, spelling mistakes, or other errors in this book.
+Please help us to fix them to produce the best book possible :) Learn here how to [contribute to this book](../book/contribute-to-this-book.md).
+{% endhint %}
+
+Welcome to the **InnerSource Patterns book**.
+
+This book contains InnerSource best practices codified in a specific format to make it easy to understand, evaluate, and reuse them. We call this reuse format a **pattern**.
+
+These patterns have been collected by the [InnerSource Commons](http://innersourcecommons.org) community over many years. The most mature of these patterns have been published in this book.
+
+In this introduction we explain [what InnerSource is](#what-is-innersource), [what a pattern is](#what-are-innersource-patterns), and [how to use these patterns](#how-can-you-use-innersource-patterns) in your organization.
+
+If you are using InnerSource in your company already and want to contribute your experiences to this book we would love to [welcome your contributions](../book/contribute-to-this-book.md)!
+
+## What is InnerSource?
+
+We define InnerSource as:
+
+> The use of open source principles and practices for software development within the confines of a company.
+
+InnerSource takes the lessons learned from developing open source software and applies them to the way companies develop software internally. As developers have become accustomed to working on world class open source software, there is a strong desire to bring those practices back inside the firewall and apply them to software that companies may be reluctant to release.
+
+For companies building mostly closed source software, InnerSource can be a great tool to help break down silos, encourage internal collaboration, accelerate new engineer on-boarding, and identify opportunities to contribute software back to the open source world.
+
+## What are InnerSource Patterns?
+
+Patterns are a way of describing a repeatable, proven solution to a problem with a context. They follow a simple form that helps people wanting to implement the solution to understand the constraints on the problem, the forces that must be balanced and the resulting context (the situation you are left with after the solution is applied).
+
+In inner sourcing, patterns can provide a way for the InnerSource Commons participants to concisely share information with each other, improving the practice of inner sourcing. Each of the patterns are divided into Title, Problem Statement, Context, Forces, and Solutions as their main sections.
```

Comment:
```suggestion
In inner sourcing, patterns can provide a way for the InnerSource Commons participants to concisely share information with each other, improving the practice of inner sourcing. Pattern descriptions are subdivided into the following sections: Title, Patlet, Problem Statement, Context, Forces, Solutions and Resulting Context.
```

Reply:
This text comes from here: https://github.com/InnerSourceCommons/InnerSourcePatterns#what-are-innersource-patterns

But again, I am happy to change it :)